share/doc/conf.py.in

```diff
@@ -8,6 +8,7 @@ source_suffix = {
     '.rst': 'restructuredtext',
     '.md': 'markdown',
```

Comment:
How is that different from this line?

Reply:
It seems that `markdown` has no parser register and according to [the sphinx docs](https://www.sphinx-doc.org/en/master/usage/configuration.html), it is possible to force a parser for a given extension

Reply:
It works on Fedora, but that is Sphinx 2 already.